src/services/postService.js

```diff
@@ -2,8 +2,14 @@ const uuid = require("uuid");
 const { throwIfError } = require('../utilities/dynamoUtilities');
 const postDAO = require("../repository/postDAO");
 
-async function createPost(userID, description, score, title){
-    const post = {class: "post", itemID: uuid.v4(), postedBy: userID, description, score, title, replies: [], likedBy: [], isFlagged: 0};
+async function createPost(userID, description, score, title, tags){
+    let tagMap = new Map();
+    if (tags){
+        for (const i of tags){
+            tagMap.set(i, true);
+        }
+    }
```

Comment:
I don't know if you tested this yet, but this is what I saw in the database for tags.

[ { "S" : "rock" }, { "S" : "rap" }, { "S" : "hip-hop" } ]

Reply:
Yes that was before I updated the tags to a map. I will test it again now that it is a map